### PR TITLE
Unlocks the Outlast stuff, gives it to vendors.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

image
image

## Why It's Good For The Game

These items were already in the game, but barely anyone uses them.

I made the Medical ones subtypes of Medical gear so they aren't a downgrade, and left the previous ones as items usable by everyone. The leather apron especially.

Part of my 'stop making items loadout exclusive' push.

## Changelog

:cl:
add: The Outlast items are no longer stuck in Loadout hell, yay!
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_gloves.dm [hunk 1]
```diff
@@ -8,9 +8,13 @@
 	donator_only = TRUE
 
 /datum/loadout_item/gloves/rubber_gloves
-	name = "Long Rubber Gloves"
+	name = "Long Rubber Medical Gloves"
 	item_path = /obj/item/clothing/gloves/longrubbergloves
-	restricted_roles = list(JOB_MEDICAL_DOCTOR,JOB_CHIEF_MEDICAL_OFFICER)
+
+/datum/loadout_item/gloves/rubber_gloves/med
+	name = "Long Rubber Gloves"
```
Comment (Cyprex) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_gloves.dm:15]:
```suggestion
	name = "Long Rubber Medical Gloves"
```

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_gloves.dm [hunk 2]
```diff
@@ -8,9 +8,13 @@
 	donator_only = TRUE
 
 /datum/loadout_item/gloves/rubber_gloves
-	name = "Long Rubber Gloves"
+	name = "Long Rubber Medical Gloves"
```
Comment (Cyprex) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_gloves.dm:11]:
```suggestion
	name = "Long Rubber Gloves"
```

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_suit.dm [hunk 3]
```diff
@@ -1,3 +1,12 @@
+/datum/loadout_item/suit/secjacket
+	name = "High-Vis Security Jacket"
+	item_path = /obj/item/clothing/suit/armor/vest/secjacket/blue
+	restricted_roles = list(JOB_WARDEN, JOB_DETECTIVE, JOB_SECURITY_OFFICER, JOB_HEAD_OF_SECURITY, JOB_CORRECTIONS_OFFICER)
+
+/datum/loadout_item/suit/leather_apron
+	name = "Leather Apron"
+	item_path = /obj/item/clothing/suit/leatherapron
```
Comment (Cyprex) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_suit.dm:8]: You removed the role restrictions for this item- I don't have an opinion on this, I'm just mentioning it(despite how the first sentence sounds- couldn't come up with a better way to word this)

Comment (theselfish) [reply modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_suit.dm:8]: I did yeah.

It doesn't have any stats, and can only hold one surgical tool at a time or now smithing tools. Beyond that it's just a regular apron.

File: modular_zubbers/code/modules/clothing/outfits/outlast.dm [hunk 4]
```diff
@@ -20,20 +20,35 @@
...
+/obj/item/clothing/gloves/latex/nitrile/longrubbergloves //Sprite by pacertest on discord
+	name = "long rubber medical gloves"
 	desc = "Pricy somewhat sterile gloves that are thicker than latex, these ones extend almost past the elbow. Excellent grip ensures very fast carrying of patients along with the faster use time of various chemical related items."
 	icon = 'modular_zubbers/icons/obj/clothing/outlast.dmi'
 	worn_icon = 'modular_zubbers/icons/mob/clothing/outlast.dmi'
 	icon_state = "longrubber"
-	clothing_traits = list(
-		TRAIT_QUICKER_CARRY,
-		TRAIT_FASTMED
-	)
```
Comment (Cyprex) [inline modular_zubbers/code/modules/clothing/outfits/outlast.dm:39]: Just for anyone that happens to stumble upon this: These aren't removed, they are still inherited from the parent /obj/item/clothing/gloves/latex/nitrile

Comment (theselfish) [reply modular_zubbers/code/modules/clothing/outfits/outlast.dm:39]: Yup. That's why I moved it up. Cleaner.

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**